src/api/grainConfig.js

```diff
@@ -4,23 +4,21 @@ import {type DistributionPolicy} from "../core/ledger/applyDistributions";
 import * as G from "../core/ledger/grain";
 import * as C from "../util/combo";
 import * as NullUtil from "../util/null";
-import {toDiscount} from "../core/ledger/policies/recent";
+import {
+  type AllocationPolicy,
+  allocationPolicyParser,
+} from "../core/ledger/policies";
 
 export type GrainConfig = {|
-  +immediatePerWeek?: number,
-  +balancedPerWeek?: number,
-  +recentPerWeek?: number,
-  +recentWeeklyDecayRate?: number,
+  +allocationPolicies: $ReadOnlyArray<AllocationPolicy>,
   +maxSimultaneousDistributions?: number,
 |};
```

Comment:
This breaks backcompat and forces an immediate upgrade for every user. Let's not do this.

Instead, please mark (thru comments) that all of the existing fields are deprecated, and leave them as optional fields.
Add the new allocationPolicies field.
Then we can upgrade all of the examples and our instance to use allocationPolicies.
Then we can remove it (ideally with a deprecation warning printed to console for the next few releases)

Reply:
Done, I agree with this approach.